### PR TITLE
Added SP 2016 to What-is-supported-SPMT.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/2193.

Source: https://spmtreleasescus.blob.core.windows.net/install/default.htm.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: migration/What-is-supported-SPMT.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ The SharePoint Migration Tool (SPMT) supports the migration of the following loc
 **On-premises fileshares**
 - Local and network fileshares
 
-**SharePoint Server 2010 & 2013**
+**SharePoint Server 2010, 2013 and 2016**
```
Comment (JohanFreelancer9) [inline migration/What-is-supported-SPMT.md:31]:
```suggestion
**SharePoint Server 2010, 2013, and 2016**
```

**Comment by scanum** [comment]
@JoanneHendrickson PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dariomws** [comment]
@JoanneHendrickson could you please check and merge? Thanks!

**Comment by dariomws** [comment]
@JoanneHendrickson could you please check and merge? Thanks!

**Pull request merged**